### ISSUE TITLE
[Bug] [Quest] The Last Line of Defense

### ISSUE THREAD

**Issue by darkwyrm88** [dalaranwow/dalaran-wow#4610]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Issue: The Frostbrood enemies (frostwyrms) are in a constant state of phasing. They do not remain in my instance long enough for me to kill them. A second or two after they phase in, they will phase out again, resetting any damage they had sustained.

**Expected behaviour**: Tell us what should happen instead
Frostbrood enemies are supposed to spawn and remain in phase for the player to kill them with the cannon to get credit for the quest.
**Steps to reproduce the problem**:

1. Start the quest.
2. Use the Cannon entity.
3. Observe the Frost Wyrm enemies.

**Include proofs for this behaviour**
I don't know what this means.
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
http://www.wowhead.com/quest=13086/the-last-line-of-defense
http://www.wowhead.com/npc=30575/frostbrood-destroyer
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TheRealestMcCoy** [comment]
https://github.com/dalaranwow/dalaran-wow/issues/4262

**Comment by Rushor** [comment]
duplicate, the issue was fixed already in https://github.com/dalaranwow/dalaran-wow/issues/4262. The fix will be tested on PTR and after some time it will be applied on the live realm.

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by catta000** [comment]
This issue persists, 2 of the 3 frostwyrms are not close enough to the cannon to be killed, but there is one frostwyrm close enough to the northern cannon. So you have to wait for his respawn after you killed it, but the quest is doable.

**Comment by seancallaway** [comment]
I just tried this today and found that the northernmost and southernmost cannons are close enough to their respective wyrms to do damage, but they are in a constant state of phase, so they keep going back to full health.

Video of the issue can be seen here: https://youtu.be/w0HGQI4iwKw
